### ISSUE TITLE
Photons disappeared

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/greenhouse-effect#348]
**Test device**
iPad 6th Gen
**Operating System**
iPadOS 16.6
**Browser**
Safari
**Problem description**
For https://github.com/phetsims/qa/issues/970.
The photons on the second screen disappeared. The temperature and sound still behaved as expected, but no photons appeared. This was seen after the sim had been open for at least 10 minutes and locked for most of that time (I had stepped out of the office for a minute). The photons on the third page still showed up. Only refreshing the whole sim fixed this, and I haven't been able to reproduce it so far.

**Visuals**
![IMG_0113](https://github.com/phetsims/greenhouse-effect/assets/41024075/12787dd3-74a7-426e-a4e9-2033d97ce7e1)

**Comment by jbphet** [comment]
@jonathanolson - I'm using sprites to display the photons, and I believe this uses the graphics coprocessor.  Is it possible that something got turned off to save power and it's state wasn't restored correctly?  Is there anything we can do about a situation like this?

**Comment by jbphet** [comment]
@arouinfar - I'd like to request your input as to whether this is blocking for an RC in case we don't hear from @jonathanolson or if there aren't any good ideas on how to address it.

**Comment by arouinfar** [comment]
> @arouinfar - I'd like to request your input as to whether this is blocking for an RC

No, I wouldn't consider this blocking, especially since @KatieWoe hasn't found a reproducible case. It seems likely that this bug exists in the published version of the sim too.

**Comment by KatieWoe** [comment]
I haven't done so in awhile, but we could do it if there was a reproducible case. As it is, I only saw this once.
